src/clj/game/cards/ice.clj

```diff
@@ -3561,14 +3584,21 @@
                  (do-net-damage 1)]})
 
 (defcard "Zed 1.0"
-  {:implementation "Restriction on having spent [click] is not implemented"
-   :subroutines [(do-brain-damage 1)
-                 (do-brain-damage 1)]
+  {:subroutines [{:label "Do 1 brain damage"
+                  :effect (req (if (spent-click-to-break-sub state run)
+                                 (resolve-ability state side (do-brain-damage 1) card nil)
+                                 (system-msg state side "does not do brain damage with Zed 1.0")))}
```

Comment:
```suggestion
                  :async true
                  :effect (req (if (spent-click-to-break-sub state run)
                                 (continue-ability state side (do-brain-damage 1) card nil)
                                 (do (system-msg state side "does not do brain damage with Zed 1.0")
                                     (effect-completed state side eid))}
```
Something like this, you'll probably have to clean it up. `do-brain-damage` is an `:async` ability, so it has to be resolved with `continue-ability` and the calling ability has to be `:async` too.

Reply:
Oh yeah, and this applies to the other Zed 1 and 2 subroutines.